src/dashboard/src/templates/ingest/grid.html

```diff
@@ -246,13 +246,16 @@ <h3>{% trans "Edit metadata" %}</h3>
           <h3>{% trans "Upload DIP" %} <img id="upload-dip-modal-spinner" src="/media/images/ajax-loader.gif" /></h3>
         </div>
         <div class="modal-body">
-          <p>{% trans "Upload the generated DIP to AtoM using the permalink of the target description." %}</p>
+          <p>{% trans "Please provide the identifier required by the AtoM or Binder SWORD V1 deposit endpoint." %}</p>
+          <ul>
+              <li><strong>AtoM:</strong>&nbsp;{% trans "In AtoM, this is the permalink or slug of the target archival description where the DIP will be attached as a child. For example, if the URL of the archival description is 'http://myAtoM.ca/newsletters-2;rad', then you would enter 'newsletters-2'." %}</li>
+              <li><strong>Binder:</strong>&nbsp;{% trans "In Binder, this is an identifier that indicates which artwork or technology record the DIP will be attached to. For example, if the DIP is being uploaded to an artwork record, and that artwork has a TMS Object ID of 123, you would enter 'ar:123'. If the DIP is being uploaded to a technology record, and that technology record has a URL in Binder of 'http://myBinder.org/technology-record/456' you would enter 'tr:456'." %}</li>
```

Comment:
Two observations:
- Our templates use two spaces for indentation.
- I think that it's preferably to annotate the whole sentence, e.g. `AtoM: ...` even if it needs to contains HTML elements that way we don't enforce a grammar structure.